### PR TITLE
Added more user related functions.

### PR DESCRIPTION
**Pull request by RobFaie**

### CODE REVIEW

File: functions/Get-User.ps1 [hunk 1]
```diff
@@ -23,12 +23,17 @@ function Get-User {
         Gets a list of all users using a connection context.
     #>
     [OutputType([PSCustomObject])]
-    [CMDletBinding()]
+    [CMDletBinding(DefaultParameterSetName = 'all')]
```
Comment (mderriey) [inline functions/Get-User.ps1:26]: `all` doesn't seem to be used on any of the parameters.

File: functions/Get-UserAuthorization.ps1 [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    .EXAMPLE
+        PS /> Get-TenableUserAuthorization -Context $context -UserUuid d64aa936-e63f-4227-a394-1699425f07e1
+
+        Returns user authorizations using a connection context.
+    #>
+    [OutputType([PSCustomObject])]
+    [CMDletBinding()]
+    Param (
+        # The UUID of the user.
+        [Parameter(Mandatory, ParameterSetName = 'uuid')]
```
Comment (mderriey) [inline functions/Get-UserAuthorization.ps1:21]: Do we need the `ParameterSetName` attribute if there's only one?

File: functions/Get-User.ps1 (hunk 1 above)
Comment (RobFaie) [reply functions/Get-User.ps1:26]: `all` is the parameter set for not providing a user id and getting all users. Defining it as the default parameter set prevent's Powershell demanding a User Id.

File: functions/Get-UserAuthorization.ps1 (hunk 2 above)
Comment (RobFaie) [reply functions/Get-UserAuthorization.ps1:21]: Nope, just a copy / paste mistake.

**Pull request merged**